tests/dragonfly/memory_test.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from redis import asyncio as aioredis
+from .utility import *
+
+
+@pytest.mark.asyncio
+@pytest.mark.opt_only
```

Comment:
Are you sure it works just like this?

Reply:
What do you mean? Did you see the changes to .github/ ci?

Reply:
Ah, read the docs wrong, it's recommended to register them, but not requried

Reply:
I tried to see how we used `slow` and saw that we don't do anything other than marking (unless I missed that)